ci/pipeline

```diff
@@ -276,7 +278,11 @@ def loop(): Unit = {
   val start = System.currentTimeMillis
   try {
     provisionHost()
-    build(buildName = utils.loopBuildName())
+    // build(buildName = utils.loopBuildName())
+
+    buildDockerAndLinuxPackages()
+    testDockerAndLinuxPackages()
```

Comment:
This will go after the loop is green.

Reply:
Y the comment?

Reply:
NOTE to anyone who decides to take charge and merge this... ☝️ that hunk should be reverted before merge